### PR TITLE
Polished Items/Feature improvement/Bug fixes for FAIR demo

### PR DESCRIPTION
**Pull request by jimmytyyang**

### CODE REVIEW

**Comment by jimmytyyang** [comment]
Feature improvement
- Reset the arm when the user steals the object from the gripper
- Human collision avoidance detection for the point nav
- Fix place target reset bug

File: examples/siro_sandbox/controllers/baselines_controller.py [hunk 1]
```diff
@@ -298,6 +313,8 @@ def get_cartesian_obj_coords(self, obj_trans):
         return rho, phi
 
     def get_geodesic_distance_obj_coords(self, obj_trans):
+        # Make sure the point is a navigatable waypoint
```
Comment (0mdc) [inline examples/siro_sandbox/controllers/baselines_controller.py:316]: Typo: `navigatable`

File: examples/siro_sandbox/controllers/baselines_controller.py [hunk 2]
```diff
@@ -315,13 +332,33 @@ def check_if_skill_done(self, observations, skill_name):
...
+        return np.array(global_T.transform_point(np.array(target)))
+
+    def get_ee_loc(self, env):
+        global_T = env._sim.get_agent_data(
+            self._agent_idx
+        ).articulated_agent.ee_transform()
+        return global_T.translation
+
+    def _check_obj_ray_to_ee(self, obj_trans, env):
+        """Cast a ray from theo object to the robot"""
```
Comment (0mdc) [inline examples/siro_sandbox/controllers/baselines_controller.py:349]: Nit: `theo`

**Review by 0mdc — APPROVED** [review]
LGTM! I would suggest to wait for @xavierpuigf's input before merging.

**Review by eundersander — APPROVED** [review]
See my comments below. Aside from them, LGTM!

**Pull request merged**